### PR TITLE
EWPP-419: Install OP publication list media type on update.

### PR DESCRIPTION
**Pull request by imanoleguskiza**
### Description

Add update path to install OP Publication lists.

### Change log

- Added: Add update path to install OP Publication lists.

- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by nagyad — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

File: oe_media.module [hunk 1]
```diff
@@ -180,3 +183,52 @@ function media_media_presave(MediaInterface $media) {
...
+ *   The config name.
+ * @param \Drupal\Core\Config\StorageInterface $storage
+ *   The configuration storage where the file is located.
+ * @param bool $create_if_missing
+ *   If the configuration entity should be created if not found. Defaults to
+ *   TRUE.
+ * @param bool $update_if_exists
+ *   If the configuration entity should be updated if found. Defaults to TRUE.
+ */
+function _oe_media_import_config_from_file(string $name, StorageInterface $storage, bool $create_if_missing = TRUE, bool $update_if_exists = TRUE) {
```
Comment (sergepavle) [inline oe_media.module:200]:
```suggestion
function _oe_media_import_config_from_file(string $name, StorageInterface $storage, bool $create_if_missing = TRUE, bool $update_if_exists = TRUE): void {
```

Comment (sergepavle) [inline oe_media.module:200]: Don't sure that we really need that third argument. Are we have a chance that post-update can be executed twice and reimporting could damage something?

**Review by sergepavle — APPROVED** [review]

File: oe_media.module (hunk 1 above)
Comment (imanoleguskiza) [reply oe_media.module:200]: There are situations where we do not want to update configuration if it already exists. For example in this case, if the site using the component already created a content typed with the same name we do not want to override it.

**Review by nagyad — APPROVED** [review]

File: oe_media.module (hunk 1 above)
Comment (sergepavle) [reply oe_media.module]: Fair enough.

**Review by sergepavle — APPROVED** [review]

**Pull request merged**